packages/cli/package.json

```diff
@@ -40,15 +41,18 @@
     "src"
   ],
   "dependencies": {
+    "chalk": "^5.4.1",
```

Comment:
chlak 5 is ESM

use chalk 4
https://stackoverflow.com/questions/70309135/chalk-error-err-require-esm-require-of-es-module